index.html

```diff
@@ -0,0 +1,162 @@
+          </a>
+        </form>
+        <!-- header middle (searchbar) -->
+
+        <!-- header right (nav-links, login/signup CTA) -->
+        <div class="flex justify-center align-items-center">
+          <!-- navigation links -->
+          <a href="/pages/wishlist-page/wishlist-page.html">
+            <span class="w-full h-full tooltip btn-icon mr-5">
+              <i
+                class="text-cyan-500 text-hover-cyan-300 fas fa-user text-3xl"
+              ></i>
+
+              <span
+                class="tooltip-text text-black text-lg tooltip-text-bottom bg-cyan-100"
+              >
+                Profile
+              </span>
+            </span>
+          </a>
+
+          <!-- login/signup CTA -->
+          <a
+            class="btn btn-square-solid btn-md text-white bg-cyan-600 btn-bg-hover-cyan-300 mr-7"
+          >
+            LOGIN
+          </a>
+
+          <!-- navigation links -->
+        </div>
+        <!-- header right (nav-links, login/signup CTA) -->
+      </div>
+    </header>
+    <!-- header -->
+
+    <!-- main section -->
+    <main
+      class="relative w-full flex align-items-center justify-center"
+      style="top: 5rem; min-height: 100vh"
+    >
+      <div style="max-width: 980px; margin: auto">
+        <div class="flex flex-col justify-center mb-10">
+          <h2 class="mb-10 text-8xl text-cyan-600">Quiz Mania</h2>
+
+          <h5 class="text-xl">
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Assumenda,
+            <br />
+            pariatur dignissimos. Optio vitae sequi reiciendis asperiores
+            cumque. repellendus unde aperiam incidunt quibusdam architecto
+            blanditiis eligendi ad ipsum voluptate, quidem odit.
+          </h5>
+        </div>
+
+        <a
+          class="btn btn-square-solid bg-cyan-600 text-white"
+          href="/pages/quiz.html"
+          >Find Quizzes</a
+        >
+      </div>
+    </main>
+    <!-- main section -->
+
+    <div class="spacer-3rem"></div>
+    <div class="spacer-3rem"></div>
+    <div class="spacer-3rem"></div>
+    <div class="spacer-3rem"></div>
+
+    <!-- footer -->
+    <footer
+      class="relative flex flex-col h-44 justify-center align-items-center w-full p-5 z-50 bg-cyan-200"
+      style="bottom: 0"
+    >
+      <div class="text-grey-700">
+        Made with
+        <span class="text-cyan-700 font-bold ml-1 mr-1">&lt;/&gt;</span> by
+
+        <a class="ml-1 text-hover-cyan-600">Ananya</a>
+      </div>
+      <div class="flex mt-3 align-items-center justify-center">
+        <a href="https://github.com/Ananyamadhu08" target="_blank">
+          <i class="fab fa-github mr-2 text-cyan-700"></i>
+        </a>
+        <a href="https://twitter.com/AnanyaMadhu27" target="_blank">
+          <i class="fab fa-twitter mr-2 text-cyan-700"></i>
+        </a>
+        <a
+          href="https://www.linkedin.com/in/ananya-madhu-74479b206/"
+          target="_blank"
```

Comment:
It is a good practice to use rel="noopener noreferrer" with target="_blank" from security point of view.